### PR TITLE
Set SQLAlchemy to use pre_ping

### PR DESCRIPTION
**Pull request by chasain**
This fixes the "server closed the connection unexpectedly" error detailed in issue #51

### CODE REVIEW

**Review by itsojon — APPROVED** [review]
LGTM. Thank you very much, @chasain!

File: server/src/main.py [hunk 1]
```diff
@@ -28,6 +28,10 @@ def init_app_without_routes(disable_csrf=False):
 
     global db
     global migrate
+    class SQLAlchemy(_BaseSQLAlchemy):
```
Comment (itsojon) [inline server/src/main.py:31]: `_BaseSQLAlchemy` needs to be set as an alias for `SQLAlchemy` during the import

**Review by itsojon — CHANGES_REQUESTED** [review]
Sorry, I was a bit overeager.

File: server/src/main.py (hunk 1 above)
Comment (chasain) [reply server/src/main.py:31]: Knew I forgot something...

**Review by itsojon — APPROVED** [review]
Thanks again!

**Pull request merged**